RetailSuiteAccounting/UserInterface/01b_Balances.md

```diff
@@ -0,0 +1,44 @@
+# Balances
+
+The *BALANCES* tab offers a T-account view of the current credit and debit situation, and the resulting balance of a selected account. The information displayed in the T-account view is read-only.
+
+
+## Balances view without account selection
+
+*Accounting > Post > Tab BALANCES*
+
+![Balances - no account selected](../../Assets/Screenshots/RetailSuiteAccounting/Book/Balances/BalancesNoAccount.png "[Balances - no account selected]")
+
+When clicking the *BALANCES* tab, a blank tab and the notice "No account selected" may be displayed. An account must be selected to display the corresponding details. If a posting is selected in the *POSTINGS* tab, the account and contra account details corresponding to the selected posting will be displayed in the *BALANCES* tab.
+
+For detailed information about accessing the balance view of an account, see [Access the balance view of an account](../Operation/02_ReviewAccount.md#access-the-balance-view-of-an-account).
+
+## Balances view with account selection
+
+*Accounting > Post > Tab BALANCES*
+
+![Balances](../../Assets/Screenshots/RetailSuiteAccounting/Book/Balances/Balances.png "[Balances]")
+
+
+- *Fiscal year XXXX/Month X/Journal X*  
+This text displays the fiscal year, month and journal currently selected. This information is read-only.
+
+  > [Info] If the fiscal year or the posting period need to be changed, click the [FY/MONTH/JOURNAL] button to display the select fiscal year window. For detailed information, see [Fiscal year](./00a_FiscalYear.md) and [Select a fiscal year](../Operation/01_SelectFiscalYear.md).
+
+- ![Left arrow](../../Assets/Icons/Back02.png "[Left arrow]") (Left arrow)   
+Click this arrow to open the *Post extras* side bar. For detailed information about searching in the *Accounting* module, see [Search in accounting](../Operation/14_SearchAccounting.md).
+
+- *Balance for account (account number) (account name)*  
+The balance for the selected **contra account** is displayed on the left side of the workspace. The balance for the selected **account** is displayed on the right side of the workspace.
+
+- *CREDIT (CURRENCY)*  
+On the left side of the T-account view, the credit is displayed.
+
+- *DEBIT (CURRENCY)*  
+On the right side of the T-account view, the debit is displayed.
+
+- *All months*  
+Click this radio button to display the balances for all months.
+
+- *Only this month*  
+Click this radio button to display the balances only for the selected month.
```

Comment:
Verweis auf Input line muss noch rein -> ggf. input line als eigenen header in 01a_Bookings oder ebenso wie die anderen Funktionen in separatem Dokument, damit man darauf verweisen kann.